samples/sensor/fdc2x1x/README.rst

```diff
@@ -0,0 +1,79 @@
+.. _fdc2x1x:
+
+FDC2X1X: Capacitance-to-Digital Converter
+#########################################
+
+Overview
+********
+
+This sample application periodically reads raw data from the FDC2X1X sensor in polling mode
+or optionally with data ready trigger. It is able to read the 12-Bit and 28-Bit, as well as
+the 2-Channel and 4-Channel versions (FDC2112, FDC2114, FDC2212, FDC2214). The 4-channel versions
+are chosen through devicetree properties. The default in this sample is the 2-channel version.
+
+
+Wiring
+*******
+
+This sample uses the FDC2X1X sensor controlled using the I2C interface.
+Connect supply **VDD** and **GND**. The supply voltage can be in
+the 2.7V to 3.6V range.
+
+Connect **SD** to a GPIO to control the Shutdown Mode.
+
+Connect Interface: **SDA**, **SCL** and optionally connect **INTB** to a
+interrupt capable GPIO.
+
+For detailed description refer to the `FDC2X1X datasheet`_
+at pages 4-5.
+
+
+Building and Running
+********************
+
+This sample outputs sensor data to the console and can be read by any serial
+console program. It should work with any platform featuring a I2C interface.
```

Comment:
You should add a note that different platforms will require custom devicetree overlays.

Reply:
Added your suggested note